### ISSUE TITLE
@Transactional annotation and TransactionalAction don't work with Secure Social plugin

### ISSUE THREAD

**Issue by rlamarche** [playframework/playframework#4371]
Please see https://github.com/playframework/play-ebean/issues/11

I consider this bug critical. I reported it 24 days ago, and no news on it. So I report it here.
How can you develop a complex application without databases transactions ?

I've found a workaround, but I think that a lot of applications using EBean ORM and SecureSocial needs transactions, but they haven't.

**Comment by jroper** [comment]
Hi @rlamarche, if you have critical issues and need action in a given timeframe, then please consider purchasing [commercial support](http://www.typesafe.com/subscription), an open source issue tracker isn't a place to get help, it's a place for everyone to contribute to the project. We're very busy with preparing for the Play 2.4.0 release, and the issue you're reporting has been around for a few years now, so it's not exactly high on our priorities.

**Comment by rlamarche** [comment]
I was not asking for support as I found a (not sexy) workaround (see https://github.com/playframework/play-ebean/issues/11).
I have reported this bug to inform other users (even if they don't pay) that if they use @Transactional and @Secured annotation on the same action, there is no transaction at all ans it can lead to corrupted data in dababase.

Indeed, no transaction = no rollback.

BTW, thank you for your work on Play 2.4, but don't forget : providing a great framework comes with a great responsibility ;)
